### PR TITLE
Ui cleanup

### PR DESCRIPTION
**Pull request by medariox**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by medariox** [comment]
**_ISSUES_**
- Groups are all lowercase, missing groups are empty instead of "None".
image

- Refresh during manual search doesn't work.

- "The" is not ignored during name sorting.*
image

- Can't change "sort by" order.*

* Only on "initial state" - changing "direction" fixes this until page refresh. Following JS errors:
image

**Comment by fernandog** [comment]
rebased

File: gui/slick/css/style.css [hunk 1]
```diff
@@ -466,57 +468,57 @@ home.mako
...
-    color: #000000;
+    color: rgb(0, 0, 0);
 }
 
 .progress-100 {
-    background-image: -moz-linear-gradient(#395f07, #2a4705) !important;
-    background-image: linear-gradient(#395f07, #2a4705) !important;
-    background-image: -webkit-linear-gradient(#395f07, #2a4705) !important;
-    background-image: -o-linear-gradient(#395f07, #2a4705) !important;
+    background-image: -moz-linear-gradient(rgb(57, 95, 7), rgb(42, 71, 5)) !important;
```
Comment (fernandog) [inline gui/slick/css/style.css]: @medariox 
release group case shouldn't be changed

**Comment by fernandog** [comment]
@medariox @labrys 

1) Options are under tab name - It should be in the right "div"
/config/notifications/
/config/general/
/config/backuprestore/
/config/anime/

2) size in failed downloads are not pretty: 3706011067  (maybe this also exists in devleop)

3) Manual search results page it not being refreshed with results

**Pull request closed without merging**